examples_tests/49.ComputeFFT/convolve.comp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#version 430 core
+
+#include <nbl/builtin/glsl/math/complex.glsl>
+
+layout(local_size_x=256, local_size_y=1, local_size_z=1) in;
+ 
+struct nbl_glsl_ext_FFT_output_t
+{
+	vec2 complex_value;
+};
+
+layout(set=0, binding=0) restrict readonly buffer SrcBuffer
+{
+	nbl_glsl_ext_FFT_output_t src_data[];
+};
+
+layout(set=0, binding=1) restrict readonly buffer KernelBuffer
+{
+	nbl_glsl_ext_FFT_output_t ker_data[];
+};
+
+layout(set=0, binding=2) restrict buffer OutputBuffer
+{
+	nbl_glsl_ext_FFT_output_t out_data[];
+};
+
+layout(push_constant) uniform PushConstants
+{
+    layout (offset = 0) uvec3 padded_dimension;
+    layout (offset = 16) uint numSrcChannels;
+} pc;
+
+void main()
```

Comment:
its wasteful to launch a kernel just to perform the complex multiplication of spectra

Reply:
you could fold forward Y FFT together with complex multiplication and then Y iFFT